packages/kit/test/apps/basics/src/routes/selection/_tests.js

```diff
@@ -0,0 +1,34 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test) {
+	test('reset selection', '/selection/a', async ({ page, clicknav }) => {
+		await page.waitForTimeout(50);
```

Comment:
Is there a reason to wait for timeout? We have tried to avoid this if possible. If it's really needed, we should add a comment explaining it..